packages/renderer/src/lib/ingresses-routes/IngressesRoutesList.svelte

```diff
@@ -0,0 +1,143 @@
+<script lang="ts">
+import { onDestroy, onMount } from 'svelte';
+
+import type { Unsubscriber } from 'svelte/store';
+import { filtered, searchPattern } from '../../stores/deployments';
+import NavPage from '../ui/NavPage.svelte';
+import Table from '../table/Table.svelte';
+import { Column, Row } from '../table/table';
+import IngressRouteColumnActions from './IngressColumnActions.svelte';
+import Button from '../ui/Button.svelte';
+import { faTrash } from '@fortawesome/free-solid-svg-icons';
+import FilteredEmptyScreen from '../ui/FilteredEmptyScreen.svelte';
+import SimpleColumn from '../table/SimpleColumn.svelte';
+import type { IngressUI } from './IngressUI';
+import { IngressRouteUtils } from './ingress-route-utils';
+import { ingresses } from '/@/stores/ingresses';
+import IngressRouteIcon from '../images/IngressRouteIcon.svelte';
+import type { RouteUI } from './RouteUI';
+import IngressRouteColumnName from './IngressRouteColumnName.svelte';
+import IngressRouteEmptyScreen from './IngressRouteEmptyScreen.svelte';
+import IngressRouteColumnHostPath from './IngressRouteColumnHostPath.svelte';
+import IngressRouteColumnBackend from './IngressRouteColumnBackend.svelte';
+
+export let searchTerm = '';
+$: searchPattern.set(searchTerm);
+
+let ingressesRoutesUI: (IngressUI | RouteUI)[] = [];
+
+const ingressRouteUtils = new IngressRouteUtils();
+
+let ingressesUnsubscribe: Unsubscriber;
+let routesUnsubscribe: Unsubscriber;
+onMount(async () => {
+  ingressesUnsubscribe = ingresses.subscribe(value => {
+    ingressesRoutesUI = value.map(ingress => ingressRouteUtils.getIngressUI(ingress));
+  });
+});
+
+onDestroy(() => {
+  // unsubscribe from the store
+  ingressesUnsubscribe?.();
+  routesUnsubscribe?.();
+});
+
+// delete the items selected in the list
+let bulkDeleteInProgress = false;
+async function deleteSelectedIngressesRoutes() {
+  const selectedIngressesRoutes = ingressesRoutesUI.filter(ingressesRoutesUI => ingressesRoutesUI.selected);
+
+  if (selectedIngressesRoutes.length > 0) {
+    bulkDeleteInProgress = true;
+    await Promise.all(
+      selectedIngressesRoutes.map(async ingressRoute => {
+        const isIngress = ingressRouteUtils.isIngress(ingressRoute);
+        try {
+          if (isIngress) {
+            await window.kubernetesDeleteIngress(ingressRoute.name);
+          } else {
+            await window.kubernetesDeleteRoute(ingressRoute.name);
+          }
+        } catch (e) {
+          console.log(`error while deleting ${isIngress ? 'ingress' : 'route'}`, e);
+        }
+      }),
+    );
+    bulkDeleteInProgress = false;
+  }
+}
+
+let selectedItemsNumber: number;
+let table: Table;
+
+/* let statusColumn = new Column<IngressUI>('Status', {
```

Comment:
Needs a basic status column. Even if status is always running, needs the icon for visual design and to match other pages.